src/formats/bear-bear2bk.ts

```diff
@@ -100,15 +108,19 @@ export class Bear2bkImporter extends FormatImporter {
 
 	private async colllectMetadata(entries: ZipEntryFile[]): Promise<{ [key: string]: Metadata; }> {
 		let metaData: { [key: string]: Metadata; } = {};
-		const infoEntries = entries.filter((entry) => entry.name === 'info.json');
-		for (let entry of infoEntries) {
+		for (let entry of entries) {
+			if (entry.name === 'info.json') {
```

Comment:
Is this backwards? I think we should be continuing if the file is _not_ `info.json`.

Reply:
Continue in this context will just skip this file, it sort of means "Continue with the loop, skipping this file"

Reply:
Right, but in this loop we're specifically looking for `info.json` files and should be skipping other files.

Actually looking at this again, there's only ever going to be one `info.json` in `entries` so we could instead use `let infoJsonEntry = entries.find((entry) => entry.name === 'info.json');`

Reply:
@tgrosinger Thanks for catching that. I tested my changes but without running npm build so it was the old version I tested with 🤦‍♂️  I fixed it now in 84bd00f.

Regarding `find()`, the `entries` array contains a flat list of all files in the backup. There are a bunch of `info.json` and related `text.md` files. The parent contains the name of the textbundle they belong to. Here an extract of the  `entries` array:
![Bildschirmfoto 2024-02-12 um 21 36 18](https://github.com/obsidianmd/obsidian-importer/assets/148483/b1476f21-fb7b-42eb-92d0-00577a1d1f6d)

Reply:
Ah, this is iterating through all the textbundles at once. Got it.